### PR TITLE
css-canvas-medical challenge

### PR DESCRIPTION
**Pull request by kvnliang4**
css-canvas-medical

### CODE REVIEW

File: css-canvas-medical/index.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        <div class="pic-content-position white-text">
+          <h2 class="fontsize50">The EMR Reimagined for Primary Care</h2>
+          <p>Can't remember the last time you talked with a patient without focusing on the EMR? Canvas lets you
+            rediscover joy in the work you do. Capture meaningful data, identify gaps in care, and move seamlessly
+            between tasks within the platform. You stay present with patients while your whole care team stays
+            up-to-speed. Be a better doctor with Canvas, designed for primary care providers with the help of
+              primary care providers.
+          </p>
+
+          <form action="get-started-placeholder.index">
```
Comment (BrettAlbright) [inline css-canvas-medical/index.html]: you don't need form elements to have buttons, you can remove all of the form tags on this page.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking good Kevin! Just a few updates:
1. Remove the Form tags from the code
2. Adjust the spacing a position of the nav menu to more closely reflect the example
3. Darken the image just a bit to more match the example image. The filter CSS property is good for that
4. Move the text and button in the `ready to learn more` row in to more closely match the example
5. Move the text in the dark gray footer  off the edge of the element as per the example.
6. Move the text in the bottom footer away from the edge of the element as per the example.

**Comment by kvnliang4** [comment]
css-canvas-medical-updated

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**